src/styles/index.css

```diff
@@ -62,6 +62,10 @@
     border-top-width: 1px;
   }
 
+  .chat-wrapper a {
+    text-decoration: underline;
```

Comment:
Of course, it will work. But I was thinking more in using the [`revert`](https://developer.mozilla.org/en-US/docs/Web/CSS/revert) CSS keyword value in order to use the default specified by the browser in its UA stylesheet by now and bypassing the Tailwind reset.

```css
.chat-wrapper a {
  color: revert;
  text-decoration: revert;
}
```

This makes the link not only to be underlined, but also use the browser defined colors for (if I'm not wrong) a link, a link:hover, a link:visited, etc.

But as you wish, we can go ahead just underlined them by now.